operator_ui/src/reducers/actions.ts

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import * as jsonapi from '@chainlink/json-api-client'
+
+export interface RedirectAction {
+  type: 'REDIRECT'
+}
+
+export interface MatchRouteAction {
+  type: 'MATCH_ROUTE'
```

Comment:
This was so that I had a typed action in the middleware. The old file is getting completely converted in https://www.pivotaltracker.com/story/show/169935509

Reply:
Ah, awesome.